packages/elements-core/src/components/TryIt/TryIt.tsx

```diff
@@ -98,20 +98,37 @@ export const TryIt: React.FC<TryItProps> = ({
 
   const [operationAuthValue, setOperationAuthValue] = usePersistedSecuritySchemeWithValues();
 
-  const servers = React.useMemo(
-    () => getServersToDisplay(httpOperation.servers || defaultServers),
-    [httpOperation.servers],
-  );
+  const servers = React.useMemo(() => {
+    const toDisplay = getServersToDisplay(httpOperation.servers || defaultServers);
+
+    if (mockUrl) {
+      toDisplay.push({
```

Comment:
Actually `getServersToDisplay` could be enhanced to return (optionally) mock server as well.

Then `ServerInfo` component could be refactored to utilise it as well. 

This would help keep all the names and options consistent throughout the codebase.

Reply:
Nice one - done